subworkflows/local/samplesheet_to_channel/main.nf

```diff
@@ -0,0 +1,60 @@
+workflow SAMPLESHEET_TO_CHANNEL {
+    take:
+    reference
```

Comment:
Could you add a comment about the expected structure of this input? Looks like it could be quite complicated, so users will need the help

Reply:
Structure for this is still not complete, I intended to fix this in future PR